### ISSUE TITLE
If a label is too long, delete button is hidden.

### ISSUE THREAD

**Issue by a-fly-fly-bird** [SkalskiP/make-sense#303]
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/70647529/209618225-80707e66-1c31-4806-9550-2d2d451285c6.png">

As you can see, it's hidden. I can not delete my annotation.

**Comment by SkalskiP** [comment]
Hi, @a-fly-fly-bird! Thanks for reporting that bug! It looks like it is not visible when the currently selected label name is too long. Let me take a look at it and work on some fixes.

**Comment by a-fly-fly-bird** [comment]
Thanks for your work.

From: Piotr Skalski ***@***.***>
Date: Tuesday, December 27, 2022 at 21:16
To: SkalskiP/make-sense ***@***.***>
Cc: Lucas Tan ***@***.***>, Mention ***@***.***>
Subject: Re: [SkalskiP/make-sense] If a label is too long, delete button is hidden. (Issue #303)

Hi, @a-fly-fly-bird<https://github.com/a-fly-fly-bird>! Thanks for reporting that bug! It looks like it is not visible when the currently selected label name is too long. Let me take a look at it and work on some fixes.

**Comment by SkalskiP** [comment]
@a-fly-fly-bird I just merged https://github.com/SkalskiP/make-sense/pull/304 into the `develop` branch. Please take a look and let me know if it works for you. 

> **Note**
> Until the next release - `1.12.0-alpha`, new changes won't be accessible at https://makesense.ai, but only on MS development version at https://develop.makesense.ai.

**Comment by SkalskiP** [comment]
@a-fly-fly-bird did you had time to test the fix?

**Comment by a-fly-fly-bird** [comment]
> @a-fly-fly-bird did you had time to test the fix?

Yeah, I run it locally, and it seems that the excess part will become three points.

<img width="324" alt="image" src="https://user-images.githubusercontent.com/70647529/209893144-ac033761-0195-4204-9103-3f470857da12.png">

**Comment by SkalskiP** [comment]
Grat! I'm closing the issue :)
